index.html

```diff
@@ -6,49 +6,208 @@
+            <a>
+                <div>
                     <img src="Images/Menu.png" alt="icon">
                     <img src="Images/Menu.png" alt="icon">
                     <img src="Images/Menu.png" alt="icon">
                 </div>
-            </nav>
-        </header>
-    </section>
-    <!-- headline section -->
-    <section>
-        <div class="headline">
-            <h1 class="h1">Hey There. I'm Kifle</h1>
+            </a>
+        </nav>
+        <!-- headline section -->
+        <section class="headline_section">
+            <div>
+                <h1 class="h1">Hey There. I'm Kifle</h1>
 
-            <h4 class="h4">I'm a Software Developer</h4>
+                <h4 class="h4">I'm a Software Developer</h4>
 
-            <p class="p">
-                I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
-don’t hesitate to contact me.            </p>
-            <!-- social media section -->
-            <ul class="image-icons">
-                <li><a href="https://github.com/kifle23"><img src="Images/GitHub.png" alt="GitHub"></a></li>
-                <li><a href="https://www.linkedin.com/in/kifle-haile-5a613761"><img src="Images/Linkedin.png"
-                            alt="Linkedin"></a></li>
-                <li><a href="https://twitter.com/KifleHaile12"><img src="Images/Twitter.png" alt="Twitter"></a></li>
+                <p class="p">
+                    I can help you build a product , feature or website. Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hesitate to contact me. </p>
+                <!-- social media section -->
+                <ul class="image-icons">
+                    <li><a href="https://github.com/kifle23"><img src="Images/GitHub.png" alt="GitHub"></a></li>
+                    <li><a href="https://www.linkedin.com/in/kifle-haile-5a613761"><img src="Images/Linkedin.png"
+                                alt="Linkedin"></a></li>
+                    <li><a href="https://twitter.com/KifleHaile12"><img src="Images/Twitter.png" alt="Twitter"></a></li>
 
-            </ul>
+                </ul>
 
-        </div>
+            </div>
+        </section>
 
+    </header>
+    <section id="resent-works">
+        <div class="heading-and-seperator">
+            <h1 class="heading">My Recent Works</h1><br />
+            <hr class="horizontal-seperator">
+        </div>
+        <div class="work-card-container">
+            <div class="img-one img"></div>
+            <h2 class="card-heading">Multi-Post Stories</h2>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
```

Comment:
- [ ] kindly consider giving this paragraph the same font family as the one provided in the Figma design this will make your project an exact duplicate of the provided design